test/outdated/goldens/circular_dependencies.txt

```diff
@@ -93,3 +80,42 @@ transitive dev_dependencies: all up-to-date
 1 upgradable dependency is locked (in pubspec.lock) to an older version.
 To update it, use `pub upgrade`.
 
+$ pub outdated --no-color --mode=null-safety
+Running in 'null safety' mode; filtering versions to those that support null safety.
+Dependencies  Current  Upgradable  Resolvable  Latest  
+foo           ?1.2.3   ?1.3.0      ?1.3.0      ?1.3.0  
+
+dev_dependencies: all null safety compliant
```

Comment:
@kwalrath Do you have ideas here. (cc @mit-mit )
Should we write 'supports null safety', 'null safety compliant' or something else.

To me 'supports null safety' sounds like it can be used with a null-safe sdk. And that is less strong than what this is.

For a package to be included it must opt-in to a null-safe language version, and all their dependencies must opt-in.

On the other hand 'compliance' might not really convey this meaning much better.

Reply:
> For a package to be included it must opt-in to a null-safe language version, and all their dependencies must opt-in.

Maybe use "fully support null safety" then?

Reply:
Done